contracts/test/helpers.js

```diff
@@ -250,6 +250,7 @@ const getAssetAddresses = async (deployments) => {
       aUSDC: addresses.mainnet.aUSDC,
       aUSDT: addresses.mainnet.aUSDT,
       AAVE: addresses.mainnet.Aave,
+      AAVE_TOKEN: addresses.mainnet.Aave,
```

Comment:
@DanielVF just wanted to run this by you. Didn't see AAVE_TOKEN yet setup for the mainnet address. I might be confused here since the mainent AAVE is ERC20 and the local AAVE (MockAave) is not ERC20. Not sure if what I've done here is the appropriate solution. 

With the above change I am setting `AAVE_TOKEN ` as a reward token in the deploy script.

Reply:
Why use AAVE_TOKEN vs using AAVE?

Reply:
In case of local environment we are marking a distinction between them:
https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/test/helpers.js#L281-L282